### PR TITLE
Update version/add headers

### PR DESCRIPTION
**Pull request by AliKhoda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,6 +39,7 @@ test:
     - test -e $PREFIX/lib/libmatio.dylib   # [osx]
     - if not exist %PREFIX%\Library\bin\libmatio.dll exit 1  # [win]
     - if not exist %PREFIX%\Library\lib\libmatio.lib exit 1  # [win]
+    - if not exist %PREFIX%\Library\include\matio.h exit 1  # [win]
```
Comment (massich) [inline recipe/meta.yaml:42]: Shall we test as well against `matio_pubconf.h` ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,7 +9,7 @@ source:
   md5: bb53fd06f7ebb74589e9dd684b1b66a6
```
Comment (massich) [inline recipe/meta.yaml]: Changing from `1.5.6` to `1.5.11` the hash should be changed accordignly
```diff
- md5: bb53fd06f7ebb74589e9dd684b1b66a6
+ md5: a79cf9d1200f37e0466062c3ce396d6b
```

Comment (AliKhoda) [reply recipe/meta.yaml]: Oops

File: recipe/meta.yaml (hunk 1 above)
Comment (AliKhoda) [reply recipe/meta.yaml:42]: The success of header copies is already checked in bld.bat.

Comment (massich) [reply recipe/meta.yaml:42]: ok

File: recipe/bld.bat [hunk 3]
```diff
@@ -30,4 +30,7 @@ copy %SRC_DIR%\visual_studio\%SLN_PLAT%\Release\libmatio.dll %LIBRARY_BIN%\
 if errorlevel 1 exit 1
 copy %SRC_DIR%\visual_studio\%SLN_PLAT%\Release\libmatio.lib %LIBRARY_LIB%\
 if errorlevel 1 exit 1
-
+copy %SRC_DIR%\src\matio.h %LIBRARY_INC%\
+if errorlevel 1 exit 1
+copy %SRC_DIR%\src\matio_pubconf.h %LIBRARY_INC%\
```
Comment (massich) [inline recipe/bld.bat]: `matio_pubconf.h` is a generated file from `.h.in`, I think that the configured output can be found in `%SRC_DIR%\visual_studio\`. Therefore,

```diff
- copy %SRC_DIR%\src\matio_pubconf.h %LIBRARY_INC%\
+ copy %SRC_DIR%\visual_studio\matio_pubconf.h %LIBRARY_INC%\
```

Comment (AliKhoda) [reply recipe/bld.bat]: Correct

**Comment by AliKhoda** [comment]
Seems alright now!

**Comment by 183amir** [comment]
Thank you

**Comment by massich** [comment]
Thank you all. 

Do you know if there's a latency until it gets to the conda-forge channel or something?

**Comment by 183amir** [comment]
once appveyor finishes on the master branch, the windows packages will be in the channel.

**Pull request merged**